src/olympia/conf/dev/settings.py

```diff
@@ -74,6 +74,6 @@
 FXA_OAUTH_HOST = 'https://oauth.stage.mozaws.net/v1'
 FXA_PROFILE_HOST = 'https://profile.stage.mozaws.net/v1'
 
-REMOTE_SETTINGS_IS_TEST_SERVER = True
+REMOTE_SETTINGS_WRITER_BUCKET = 'staging'
```

Comment:
we can probably just change the default in settings_base to `staging` and drop the overrides in prod and stage too

Reply:
https://github.com/mozilla/addons-server/pull/22724/commits/addf8ff91cd776ed2463373c5240343cf483cdae